webapp/src/App.tsx

```diff
@@ -38,42 +53,46 @@ function App() {
       redirectUri={window.location.origin}
     >
       <div className="App">
-        <MainScreen />
-        <Routes>
-          <Route path="/" element={<HomePage />} />
-          <Route path="home" element={<HomePage />} />
-          <Route path="recipe" element={<Recipe />} />
-          <Route path="recipe-list" element={<RecipeList />} />
-          <Route path="profile" element={<UserProfile />}>
-            <Route path=":userName" element={<UserProfile />} />
-          </Route>
+        <Provider value={client}>
+          <MainScreen />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="home" element={<HomePage />} />
+            <Route path="recipe" element={<Recipe />} />
+            <Route path="recipe-list" element={<RecipeList />} />
+            <Route path="profile" element={<UserProfile />}>
+              <Route path=":userName" element={<UserProfile />} />
+            </Route>
 
-          {/* Leave this route, but don't necessarily display it once you have real features. */}
-          <Route path="/created_by" element={<CreatedBy />}>
-            {/* Using the same element since it's handling both. Normally you'd probably have a separate one */}
-            <Route path=":displayName" element={<SoundEffectTWO />} />
+            {/* Leave this route, but don't necessarily display it once you have real features. */}
+            <Route path="/created_by" element={<CreatedBy />}>
+              {/* Using the same element since it's handling both. Normally you'd probably have a separate one */}
+              <Route path=":displayName" element={<SoundEffectTWO />} />
+              <Route
+                path="/created_by/yellow-power-ranger/:displayName"
+                // eslint-disable-next-line indent
+                // eslint-disable-next-line indent
```

Comment:
Huh that's weird, maybe we should change that rule if we like it. I don't think you need it twice in a row but oh well